gitstatus.py

```diff
@@ -12,7 +12,7 @@
 
 error_string = error.decode('utf-8')
 
-if 'fatal: Not a git repository' in error_string:
+if 'fatal: Not a git repository' in error_string.lower():
```

Comment:
`Not` -> `not`

Reply:
as @peikk0 said 'fatal: Not a git repository' should be 'fatal: not a git repository'